base_user_gravatar/README.rst

```diff
@@ -6,35 +6,27 @@
 Synchronize Gravatar image
 ==========================
 
-Use you Gravatar image in to your Odoo
+Use your Gravatar image in to your Odoo
 
 Usage
 =====
 
 To configure this module, you need to:
 
-* go to user profile
+* edit user profile
 * press "Get Gravatar image" button
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
    :target: https://runbot.odoo-community.org/runbot/149/8.0
 
-.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
-.. branch is "8.0" for example
-
-For further information, please visit:
-
-* https://www.odoo.com/forum/help-1
-
```

Comment:
Yes I know but I usually remove it too! I think it's an unnecessary info IMHO. Maybe we can move this discussion to https://github.com/OCA/maintainer-tools What do you think?